lib/spack/spack/build_systems/autotools.py

```diff
@@ -151,13 +151,25 @@ def configure(self, spec, prefix):
         options = ['--prefix={0}'.format(prefix)] + self.configure_args()
         inspect.getmodule(self).configure(*options)
 
+    def build_targets(self):
```

Comment:
@alfredo-gimenez Reason above :smile: I would leave `configure_args` as is and change only `build_targets` and `install_targets`.